libflux/src/flux/parser/strconv.rs

```diff
@@ -80,10 +80,10 @@ pub fn parse_regex(lit: &str) -> Result<String, String> {
     if lit.len() < 3 {
         return Err(String::from("regexp must be at least 3 characters"));
     }
-    if lit.chars().next().unwrap() != '/' {
+    if !lit.starts_with('/') {
         return Err(String::from("regexp literal must start with a slash"));
     }
-    if lit.chars().last().unwrap() != '/' {
+    if !lit.ends_with('/') {
```

Comment:
Good change. Looks much cleaner. I didn't know that this function existed.

Reply:
I _know_ the function exists and still forget to use it regularly. I've come to appreciate `clippy` for this reason. :)